README.md

```diff
@@ -26,7 +26,7 @@ and submit a pull request on GitHub. It is a good idea to set up an issue
 describing the feature you are working on to give the ITUGLIB team a heads up
 on your contribution.
 
-You must fill out a [Contributor License Agreement](https://github.com/ituglib/gmake/CONTRIBUTING.md)
+You must fill out a [Contributor License Agreement](CONTRIBUTING.md)
```

Comment:
Or `https://github.com/ituglib/gmake/blob/main/CONTRIBUTING.md`, but why?

Reply:
I think it needs blob/main. Full qualification is needed because the content is not always presented appropriately relative by github

Reply:
It works without

Reply:
It works without and independent of branch/clone/fork that way, so this should be better.